### ISSUE TITLE
Cannot find titles with additional characters

### ISSUE THREAD

**Issue by hendersonjeff** [SchizoDuckie/DuckieTV#1210]
**What build of DuckieTV are you using (Standlone / Chrome Extension (New Tab / Browser Action))**
Standalone...

**What version of DuckieTV are you using (Stable 1.1.x / Nightly yyyymmddHHMM)**
Stable 1.1.5 ...

**What is your Operating System (Windows, Mac, Linux, Android)**
Windows...

**Describe the problem you are having and steps to reproduce if available**
Add "Marvel's Spider-man" as a show.
No auto-download.
Try "Find a torrent".
No matter how you spell it, there are never any shows found for the episodes. At best, it finds "Ultimate Spider-man". I am able to download it without DuckieTV. ...

**Attach any DuckieTV statistics or Developer Console logs if available**
...

**Comment by garfield69** [comment]
I have no problems finding episodes for this series.
the torrent scene uses `Marvels Spider-Man SxxExx` as the torrent naming template which Dtv uses as a default naming scheme for the *Marvel's Spider-Man* tv show.

as an example, I get get hits for `Marvels Spider-Man s02e15` from 1337x, Idope, Limetorrents, RarBG, ThePirateBay, TorrentDownloads, TorrentZ2 and Zooqle Search Engines (SE).
![image](https://user-images.githubusercontent.com/6933240/65861349-98523700-e3c8-11e9-8e66-0265a5e291c7.png)
Note, the screenshot above is using the manual search multi-SE dialogue. The Auto-Download Service only uses a single SE, so if you have set the default SE to one that does not have this series then the auto-download will not work. But you can set the series settings for that show to use another SE if that is required. 

But the SE on Dtv 1.1.5 are behind the times and have trouble downloading the magnets and .torrents from some sites, which may be the issue for you. For example, downloading from 1337x does not work on Dtv 1.1.5 because the 1337x web site changed their download links 3 months or more ago.
Or perhaps your local ISP is blocking many of the SE and that could be a problem too.

I recommend upgrading your Dtv Standalone 1.1.5 to the latest Nightly from https://github.com/DuckieTV/Nightlies/releases which has updated Search Engines that should all be working.

let me know how you go.

If you are still not getting any results, then we can dig deeper, because unlike Dtv 1.1.5, the Nightly comes with the Development Tools module, which will help us do some research with you to figure out what might be going on.

**Comment by hendersonjeff** [comment]
I tried the latest Nightly, for me the same results:
![image](https://user-images.githubusercontent.com/50558992/65958499-be012f80-e40c-11e9-88b0-eb6149ec4cdf.png)
![image](https://user-images.githubusercontent.com/50558992/65958533-d8d3a400-e40c-11e9-99c0-7fbd0e5098a1.png)
![image](https://user-images.githubusercontent.com/50558992/65958576-f0ab2800-e40c-11e9-9515-a24390dd919c.png)
It will also find Marvels Agents of S.H.I.E.L.D.

Now I find that I can find *some* episodes like this...."spider-man s02e", but there are a lot more out on Torrentz2.eu and other sites.
Thats's it, only RarBG and only 5 of 17 for the season.

**Comment by hendersonjeff** [comment]
![image](https://user-images.githubusercontent.com/50558992/65958845-a0809580-e40d-11e9-804b-dfa80b5acc9d.png)
Can find Ultimate Spider-man (really old show)
![image](https://user-images.githubusercontent.com/50558992/65958889-bee69100-e40d-11e9-8a66-5f4efc667104.png)
Can find Marvel's Agents of S.H.I.E.L.D.
....but not "Marvels Spider-man S02Enn" where nn is an number 01-17 and also not the default provided by Duckie "Marvels Spider-Man s02e17 marvel spider".
Adding an apostrophe (Marvel's) or taking away the dash (spiderman) doesn't help.

**Comment by hendersonjeff** [comment]
Since Torrentz2.eu is in my Duckie torrents search list, I would expect at the very least for my torrents found list to include the following (but does not)...
![image](https://user-images.githubusercontent.com/50558992/65959359-fefa4380-e40e-11e9-86f7-f0f9d544a9cc.png)

**Comment by 2635599** [comment]
even though you have it as marvel's in your main post do you actually have it that way in dtv or are you missing the apostrophe? i ask cause when you entered your search into torrenz2 you did not include the apostrophe. it must be in what you enter into dtv to search for to get the right spider-man.

**Comment by garfield69** [comment]
@hendersonjeff  the reason you cannot see the results you are expecting is because you have set a default **Minimum Size** filter of 600MB ;-)
so visit the Dtv > Settings > torrent search engine side-panel  and scroll down to the Minimum Size section and remove the 600 and Save.
OR
on the torrent search dialogue un-tick the Minimum Size check box (but this will not fix the auto-download only the manual search)
OR
override the default  for just this series, by editing the spider-man series settings and loading a smaller value
![image](https://user-images.githubusercontent.com/6933240/65993384-a3eb4e00-e4ed-11e9-9421-c313c3d1d22c.png)
and your problem should be solved.

**Comment by hendersonjeff** [comment]
Thanks for helping me figure that out, I am amazed that this is the only show of several that every single file is less than 600 MB. Another problem I discovered is that since I thought it might be something with special characters I was adding search for "spider" and "marvel" that just compounded my problems and kept the filter from finding anything. Now that I've cleared that and reduced the minimum size, things are working as they should be. I hate it when I'm the problem and the software is not. I used to test software for a living. Sometimes it took a lot of trial and error to get the man out of the error loop. :)
